lib/elm-test.js

```diff
@@ -133,6 +133,17 @@ function main() {
       'Use a custom path to an Elm executable (default: elm)',
       undefined
     )
+    .addOption(
+      new Option(
+        "--output <output>",
+        "Allowed for compatibility with `elm make`. The only supported value is /dev/null, which elm-test ignores since it does not produce any output files."
+      ).argParser(function(value) {
+          if (value !== "/dev/null") {
+            throw new InvalidOptionArgumentError("Only /dev/null is allowed.");
+          }
+        })
```

Comment:
Another way of writing this is `.choices(['/dev/null'])`. Is it better?

Error message now:

```
error: option '--output <output>' argument 'strawberry' is invalid. Only /dev/null is allowed.
```

Error message with `.choices`:

```
error: option '--output <output>' argument 'strawberry' is invalid. Allowed choices are /dev/null.
```

Reply:
> Another way of writing this is .choices(['/dev/null']). Is it better?

Yup! I totally missed that in the commander docs, but it makes a lot of sense to use the built-in way to validate the attribute.

💭 With that said, I can think of one reason for keeping a custom option, which ties into your previous comment. Since we are omitting the new option in `--help` users might have a hard time figuring out why `--output` behaves the way it does (assuming someone actually notices it!).

With that in mind, we may want to customize the error message a bit to elaborate on why it exists and why it only accepts `/dev/null`.

No strong preference, just thought I'd share the idea before switching to `.choices`.

Reply:
I think we can summarize then:

1. Remove the help text for `--output` altogether since it cannot be seen.
2. Use `.argParser` like now with an elaborate error message.
	- Please use `=>` function instead of `function`.
	- This way there’s no need for the “Emacs comment” I asked for – it’ll be in the error message.
3. Duplicate this test and add `--output /dev/null` to it: https://github.com/rtfeldman/node-test-runner/blob/c493674c278e7a2d6f66742b68296af4eb9d8dbd/tests/flags.js#L179-L182
4. Run Prettier